tests/hypothesis_strategies/alternative_creation/test_kernels.py

```diff
@@ -0,0 +1,40 @@
+"""Test alternative ways of creation not considered in the strategies."""
+
+from fractions import Fraction
+
+import hypothesis.strategies as st
+import pytest
+from hypothesis import given
+
+from baybe.kernels import MaternKernel
+
+# Strategies for creating (hopefully different) representations of the fractions that
+# can be used to create the MaternKernel
+one_half_fraction_strategy = st.fractions(min_value=0.5, max_value=0.5)
+three_halves_fraction_strategy = st.fractions(min_value=1.5, max_value=1.5)
+five_halves_fraction_strategy = st.fractions(min_value=2.5, max_value=2.5)
+
+# Putting all of the fraction strategies together
+fraction_strategy = st.one_of(
+    one_half_fraction_strategy,
+    three_halves_fraction_strategy,
+    five_halves_fraction_strategy,
+)
+
+
+@given(fraction_strategy)
+def test_different_fractions(nu: Fraction):
```

Comment:
whats the point of this test and why is it in alternative construction?

Reply:
In theory, users can also provide other fractions like 6/4 which are mathematically valid choices. I thus wanted to test whether the kernel is robust against arbitrary fractions which are de facto just 1/2, 3/2, 5/2. If this is unnecessary, we can also just remove it.

Reply:
I also don't get the point to be honest. Seems like "mit Kanonen auf Spatzen geschossen" to me. Having the test is fine I think, but using a normal parametrization containing the three fraction fully suffices. (Remember: your test now runs like 100 times or so with repeated input)

Reply:
Ok, I will change this to a test similar in style to the others and just test for the three `Fraction` objects.

Reply:
Changed. Since I am not sure whether this is no as you would prefer resp. if we should keep the test in general, I will not resolve now until you are happy. Will however resolve other threads related to these tests such that we have a single point of discussion.

Reply:
I think this is fine now! Will resolve